ONSdigital/eq-survey-runner
# Moved default schemas to 'en' directory

### What is the context of this PR?
- Moved schemas from app/data to data/en directory within survey-runner 
- Removed some logic from the code
- Changed tests to match the new directory

### How to review 
- Check I haven't missed any schemas/references to schemas
- Check to see if there are any other dependencies
- Run tests/lint
